docs/user-guide/kubectl/kubectl_describe.md

```diff
@@ -18,11 +18,6 @@
 If you are using a released version of Kubernetes, you should
 refer to the docs that go with that version.
 
-<!-- TAG RELEASE_LINK, added by the munger automatically -->
```

Comment:
I am like a 5 years old child looking to a new toy. This is the real cause? It would not cross my mind to blaim this for it. Thank you very much.